src/igvfd/tests/test_audit_measurement_set.py

```diff
@@ -142,7 +142,7 @@ def test_audit_inherit_related_multiome(
     testapp.patch_json(
         measurement_set_multiome['@id'],
         {
-            'seqspec': 'https://github.com/IGVF/seqspec/blob/main/assays/10x-ATAC/spec.yaml',
+            'protocol': 'https://www.protocols.io/view/example_protocol',
```

Comment:
why protocol is introduced here?

Reply:
This test was created to test that multiome measurement sets are inheriting the other's audits. The seqspec audit was used to test this, but since it has been removed in this PR I used the protocol audit instead, which is basically the same type of audit checking for a missing property.